pkg/pb/contact.proto

```diff
@@ -0,0 +1,32 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+syntax = "proto3";
+option go_package = "pb";
+
+package contact;
+
+import "gogo.proto";
+
+service ContactNode {
+    rpc Pingback(PingbackRequest) returns (PingbackResponse);
+}
+
+service ContactSatellite {
+    rpc Ping(PingRequest) returns (PingResponse);
+}
+
+message PingRequest {
+    node.Node sender = 1;
```

Comment:
I think there might be some unnecessary fields in `Node` and maybe we should pull them out. Just a thought, it's your call

Reply:
true. we only need the nodeAddress and nodeID right?

Reply:
I think so